File: backend/app/api/health.py
from fastapi import APIRouter

from app.core.config import settings

router = APIRouter()


@router.get("/health")
def health_check() -> dict[str, object]:
    return {
        "status": "ok",
        "app": settings.app_name,
        "environment": settings.app_env,
        "tenant": settings.default_tenant_id,
        "services": {
            "api": "up",
            "database": "configured",
            "redis": "configured",
        },
    }

File: frontend/src/api/health.ts
import { apiClient } from './client'

export interface HealthResponse {
  status: string
  app: string
  environment: string
  tenant: string
  services: {
    api: string
    database: string
    redis: string
  }
}

export const fetchHealth = async (): Promise<HealthResponse> => {
  const { data } = await apiClient.get('/health')
  return data
}
